src/main/java/com/google/sps/servlets/EventServlet.java

```diff
@@ -60,7 +60,23 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       getEvents(request, response, userService);
       return;
     }
-    // TODO: parse "events/{event_id}" here.
+    
+    String[] pathParts = pathName.split("/");
+    String eventId = pathParts[1];
+
+    if (pathParts[2].equals("join")) {
```

Comment:
Please: [Keep URLs verb free](https://opensource.zalando.com/restful-api-guidelines/#141).
GET is also meant to only read resources (https://opensource.zalando.com/restful-api-guidelines/#get) if you want to add something use POST or PUT.
Here, I would do a POST to /events/{event-id}/joined-users for example